scripts/run_check_release_notes

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+.+\*\(.+\)\*"
+if [ $? != "0" ]
+then
+	echo "The release notes were not extended correctly."
+	echo "Please make sure you added at least one line which says *(your name)* to"
```

Comment:
I would replace “says” with “states”/“mentions” here or reformulate the sentence a little bit:

 > Please make sure you add at least one line describing you contribution followed by the text “*(your name)*”.

. There is also a minor typo at the end of the current sentence:

> …which says *(your name)* to**o.**

.